### ISSUE TITLE
Creating inline citations using numeric citation styles like Vancouver and IEEE

### ISSUE THREAD

**Issue by cclpeng** [citation-js/citation-js#190]
Hello, 

I was wondering if there's a proper way to use citation-js with a citation style like 'vancouver' so that if I create a few inline citations, their numbers will increment in the order that they were inserted to the document. When I add all entries into the Cite object for the bibliography, it will put them in order, but I don't know how to make the format type "citation" rather than "bibliography" also in order

```
[1]
[1]
[1]

Bibliography 
[1]
J. A. Hobson, “Sleep is of the brain, by the brain and for the brain,” Nature, vol. 437, no. 7063, pp. 1254–1256, Oct. 2005, doi: 
          10.1038/nature04283.
[2]
U. K. Tomar and S. C. Gupta, “Somatic embryogenesis and organogenesis in callus cultures of a tree legume ? Albizia richardiana King,” 
          Plant Cell Reports, vol. 7, no. 1, pp. 70–73, Jan. 1988, doi: 10.1007/bf00272982.
[3]
U. Gianelli et al., “Lymphomas of the Bone: A Pathological and Clinical Study of 54 Cases,” Int J Surg Pathol, vol. 10, no. 4, pp. 257–266, 
          Oct. 2002, doi: 10.1177/106689690201000403.

```

Thanks so much for the help!

**Comment by larsgw** [comment]
I attempted this in c5c3e8c (see #119) but this failed (#141). I will try again tomorrow if I have time, I quickly looked at the docs again and it seemed less difficult than I thought (but we'll see).

**Comment by larsgw** [comment]
Fix released in `v0.6.7`.

**Comment by cclpeng** [comment]
This works so well, thanks for putting in all the time to make it work! 

For anyone that wants an example, I use what larsgw added kind of like this:
```
...
data1.id = '1';
citeObj.add(data1);
...
data2.id = '2';
citeObj.add(data2);
citeObj.format('citation', {
            format: 'html',
            template: 'iee',
            lang: 'en-US',
            entry: [ {id: '1', label: 'page', locator: 3}, {id: '2'} ], 
});

data3.id = '3';
citeObj.add(data3);
citeObj.format('citation', {
            format: 'html',
            template: 'iee',
            lang: 'en-US',
            entry: [ {id: '1'}, {id: '3', label: 'section', locator: 15} ], 
            citationsPre: ['1', '2'],
}); 
```
